src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -1576,7 +1594,8 @@ def aks_remove_dev_spaces(cmd, client, name, resource_group_name, prompt=False):
             raise CLIError(ae)
 
 
-def _update_addons(instance, addons, enable, workspace_resource_id=None, no_wait=False):
+# pylint: disable=line-too-long
```

Comment:
Break the line instead of using pylint disable.

Reply:
Fixed.